### PR TITLE
Parent-dash-support-single-page-animated

### PR DESCRIPTION
**Pull request by Robert-Lark**
Description
This is the second of three iterations on the support page of the parent dashboard. In this iteration both the FAQ and Contact us forms are displayed on the real estate of the support page. In this iteration, the FAQ appears on the screen when the user clicks the FAQ button and the contact form appears when the user clicks the contact us button. 

GSAP was added to the package.json to handle animations.

Fixes #

What work was done?
added contact form and FAQ to support landing page using gsap animations. 
Why was this work done?
to allow the user to Access resources on FAQ and to contact the staff at Story Squad.
What feature / user story is it for?
As a parent, I would like a clear dashboard that allows for support like a contact page.
Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, but not tested (may need new tests)

## Has This Been Tested

- [x] No

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "eslint": "^6.6.0",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.20.6",
+    "gsap": "^3.6.1",
     "prettier": "^2.1.1",
```
Comment (Robert-Lark) [inline package.json:25]: gsap library added to package.json to handle animations

**Comment by Robert-Lark** [comment]
This code is not to be merged in until after we hear back from the stakeholders as to which user flow they are wanting to implement.

File: package.json (hunk 1 above)
Comment (cearc-sara) [reply package.json:25]: Thank you for the explanation on what this was for.

File: src/components/pages/SupportPage/renderSupportPageAnimate.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+import emailjs from 'emailjs-com';
+import { useSelector } from 'react-redux';
+
+const { Panel } = Collapse;
+
+function RenderSupportPageAnimate({ success }) {
+  //access the parents information in the reducer so we can pre-populate the form with their name & email.
+  const userInfo = useSelector(state => state.parent);
+  //This function takes care of sending the data captured in the
+  // form to a dedicated email address.
```
Comment (cearc-sara) [inline src/components/pages/SupportPage/renderSupportPageAnimate.js:14]: great inline comments left throughout the code base. well done!

File: src/styles/less/SupportPage.less [hunk 3]
```diff
@@ -1,13 +1,135 @@
 @import '../less/variables.less';
 
+
+// // contact us modal
```
Comment (cearc-sara) [inline src/styles/less/SupportPage.less:4]: make sure to leave comments as to why this is commented out

**Review by cearc-sara — APPROVED** [review]
Looks great just review the comments I made inline and make the adjustments where they are needed

**Comment by Robert-Lark** [comment]
Unused Iteration

**Pull request closed without merging**